### PR TITLE
pre-commit, black, isort

### PR DESCRIPTION
**Pull request by sbidoul**
This one requires a manual intervention to fix a pylint error (missing super).

### CODE REVIEW

**Comment by sbidoul** [comment]
@grindtildeath, I'm assigning to you as you made the migration.

**Comment by grindtildeath** [comment]
@sbidoul Sorry I missed your ping, here's the fix https://github.com/acsone/server-env/pull/1

**Comment by sbidoul** [comment]
@grindtildeath the lint phase is red now. Can you re-do your commit with a 'pre-commit run --all-files'?

**Comment by grindtildeath** [comment]
@sbidoul My bad, fixed now.

File: .travis.yml [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
-
-
 install:
-  - pip install -q unidecode
-  - pip install unicodecsv
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  # Requirements to test server_environment modules
-  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
```
Comment (yvaucher) [inline .travis.yml:32]: The addition of `running_env` is necessary for `server_environment` module.

File: .travis.yml [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
-
-
 install:
-  - pip install -q unidecode
-  - pip install unicodecsv
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
```
Comment (yvaucher) [inline .travis.yml:42]:
```suggestion
  - travis_install_nightly
  # Requirements to test server_environment modules
  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
```

**Review by yvaucher — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (yvaucher) [reply .travis.yml:42]: I have to check if now it's odoo_serverrc instead.

Comment (yvaucher) [reply .travis.yml:42]: I confirm it didn't change https://github.com/OCA/maintainer-quality-tools/blob/master/travis/test_server.py#L270

File: .travis.yml [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
-  # Requirements to test server_environment modules
-  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
-  - ln -s ${TRAVIS_BUILD_DIR}/server_environment_files_sample ${TRAVIS_BUILD_DIR}/server_environment_files
```
Comment (yvaucher) [inline .travis.yml:33]: This on the other hand might not be necessary anymore.

**Comment by sbidoul** [comment]
Feel free to take over this PR, I've no time to work on it short term.

**Comment by yvaucher** [comment]
@sbidoul Ok, will do

**Comment by yvaucher** [comment]
superseeded by https://github.com/OCA/server-env/pull/36

**Pull request closed without merging**